js_modules/dagster-ui/packages/ui-core/src/assets/AssetsOverview.tsx

```diff
@@ -403,7 +405,24 @@ const SectionName = styled.span`
   font-size: 12px;
 `;
 
-const AssetCount = styled.span`
-  color: ${Colors.textLight()};
-  font-size: 14px;
+const TextOverflowStyle: React.CSSProperties = {
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+};
+
+const GridStyle = styled.div`
+  padding: 16px 12px;
+
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  @media (max-width: 768px) {
+    grid-template-columns: 1fr;
+  }
+`;
+
+const GridItem = styled.div`
+  padding: 6px 12px;
+  display: grid;
+  grid-template-columns: auto auto;
```

Comment:
you don't need display grid here if you're just going to `auto auto`;
display flex is fine.

Reply:
(unless you have an actual grid -- more than 1 row, which I think you dont here?)

Reply:
Kind of a marginal difference, but with this grid we see:
<img width="270" alt="image" src="https://github.com/dagster-io/dagster/assets/29110579/25dbb626-1c35-439a-a9e2-73e9c348c8ff">


And with flex instead, we see:
<img width="350" alt="image" src="https://github.com/dagster-io/dagster/assets/29110579/83b9561e-1e5e-4c36-9285-e1e98e82e256">

Reply:
The grid version seems to display a bit more cleanly but the flexbox version displays more content?

Reply:
oh I think in this case we want the left side to be able to shrink if we go with flex

Reply:
It does truncate.. not sure if I need to add other CSS properties. I switched this to use flexbox though